ui/src/app/applications/components/filter/filter.scss

```diff
@@ -112,6 +117,7 @@ $slate: #191826;
         align-items: center;
         height: 1.5em;
         margin-bottom: 0.5em;
+        margin-top: 0.5em;
```

Comment:
I'm not sure if I like how the way the filters all shift down once you enable one of the filters.  eg. check one of the the Health Status filters, then the filters shift down to accommodate the Clear All button. And you want to quickly check one of the other Health Status filter, so you have to process what happened.

What do you think about this behavior instead. Try changing it to this to see what I mean:

        margin-bottom: 0.0em;
        margin-top: -1.5em;

        .argo-button {
            margin-left: 25%;
            margin-top: 1.2em;

Reply:
I agree. I ended up adding just a little bit more space between the nav items and CLEAR ALL button so that it doesn't look as cramped:

<img width="259" alt="CleanShot 2023-02-07 at 13 43 40@2x" src="https://user-images.githubusercontent.com/6250584/217373446-f6e0ec37-c367-4158-a0db-9b3945d7e613.png">
<img width="247" alt="CleanShot 2023-02-07 at 13 43 33@2x" src="https://user-images.githubusercontent.com/6250584/217373467-aaed7e59-db02-4e2e-8abf-3318cd494db6.png">